src/pages/en/core-concepts/astro-components.md

```diff
@@ -215,12 +218,21 @@ const name = "Astro"
 
 ### Slots
 
-The `<slot>` element is a placeholder for HTML which will be passed in from outside of the component by "children" (as they are called in React or Preact). Slots are a way of passing data into an Astro component and are useful when you will want to reuse an "outer" component, rendered "around" data coming from an external source.
-
-A component that uses the `<slot />` element can be thought of as a reusable "wrapper" around other content, and this pattern is the basis of an Astro [Layout component](/en/core-concepts/layouts).
+Astro components can accept HTML content in your component template. This "child" content will be injected into its `<slot />` element.
 
+This pattern is the basis of an Astro layout component: an entire page of HTML content can be “wrapped” with `<Layout></Layout>` tags and sent to the Layout component to render inside of common page elements.
```

Comment:
```suggestion
```
Move this to a better location under the code block

Reply:
Yes, I had been thinking about this.